patches/api/0468-Deprecate-Bukkit-getLogger.patch

```diff
@@ -0,0 +1,36 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: powercas_gamer <cas@mizule.dev>
+Date: Thu, 4 Apr 2024 16:57:58 +0000
+Subject: [PATCH] Deprecate Bukkit#getLogger
+
+
+diff --git a/src/main/java/org/bukkit/Bukkit.java b/src/main/java/org/bukkit/Bukkit.java
+index db46e3233edf18bd91a6da8612ccc9f0080d5c17..ab532ab39a373cc0e832e23528eb95597f66f334 100644
+--- a/src/main/java/org/bukkit/Bukkit.java
++++ b/src/main/java/org/bukkit/Bukkit.java
+@@ -1056,8 +1056,10 @@ public final class Bukkit {
+      * Returns the primary logger associated with this server instance.
+      *
+      * @return Logger associated with this server
++     * @deprecated This logger doesn't have a plugin prefix and therefore makes it hard for people to debug. Use {@link org.bukkit.plugin.Plugin#getLogger()}
```

Comment:
Somebody who can tonguetangulate better than me would be appreciated, but, generally;
This logger is owned by the Minecraft server software, for logging plugin messages, you should use a Logger from within your own contexts, i.e. Plugin#getLogger(), or create a logger for your Class (i.e. slf4j) in order to retain contextual information which is used for identifying the source of messages